### DISCUSSION TITLE
No decimal point on IOS numpad

### DISCUSSION THREAD

**Discussion by NielsDiefenthaeler** [janstehno/cotw-companion#83] [Issues] [answered]
The Issue occurs when entering any number value on an IOS Device. For example when you want to enter the Trophy score or the weight of an animal in the catch book you get the IOS numpad which, for some reason gives you a comma instead of a point. Therefore you can´t enter the precise score or whatever, but have to either copy and paste a point in or just round the number.

**Comment by janstehno** [comment]
Hey, thank you for submitting the issue. I had no idea the decimal point is not there. Do not own an IOS device, sorry. I will look into it. I will adjust the checks for the decimal numbers so it works with comma too.

**Comment by janstehno** [comment, marked as answer]
This should be fixed in the latest update. Please, let me know if it does not work.

**Reply by NielsDiefenthaeler** [reply]
Just checked it out. Works perfectly :)

**Reply by janstehno** [reply]
Glad it works. Thank you for the feedback.
